backend/TreeOfAKind.Infrastructure/TreeOfAKind.Infrastructure.csproj

```diff
@@ -8,6 +8,14 @@
     <RootNamespace>TreeOfAKind.Infrastructure</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <NoWarn>1701;1702;8632</NoWarn>
```

Comment:
So that you can write? in the project without NRT without warnings

Reply:
Why would there be warnings in the first place?

Reply:
I mean NRT is disabled in an infrastructure project so TYPE? is the same as TYPE and it produces a warning that ?  is redundant